solr/core/src/test/org/apache/solr/cloud/TestTolerantUpdateProcessorRandomCloud.java

```diff
@@ -220,12 +220,12 @@ public void testRandomUpdates() throws Exception {
             final String q;
             if (causeError) {
               // even though our DBQ is gibberish that's going to fail, record a docId as affected
-              // so that we don't generate the same random DBQ and get redundent errors
-              // (problematic because of how DUP forwarded DBQs have to have their errors deduped by
+              // so that we don't generate the same random DBQ and get redundant errors
+              // (problematic because of how DUP forwarded DBQs have to have their errors deduced by
```

Comment:
could "deduplicated" rather than "deduced" have been the intended meaning here?
```suggestion
              // (problematic because of how DUP forwarded DBQs have to have their errors deduped by
```